### ISSUE TITLE
Stable or Nightly doesn't work on M1 macOS - dyld issue with homebrew

### ISSUE THREAD

**Issue by abhishiv** [PostgREST/postgrest#1785]
### Environment

* Operating system: macOS big sur 11.2.3
* 

### Description of issue

Postgres is installed via homebrw
```
⋊> ~/w/g/gratico on master ⨯ brew info  postgres                        23:43:26
Warning: Treating postgres as a formula. For the cask, use homebrew/cask/postgres
postgresql: stable 13.2 (bottled), HEAD
Object-relational database system
https://www.postgresql.org/
/opt/homebrew/Cellar/postgresql/13.2_1 (3,218 files, 43.4MB) *
  Poured from bottle on 2021-04-02 at 02:13:17
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/postgresql.rb
License: PostgreSQL
```

Error I get is following
```
./bin/postgrest
dyld: Library not loaded: /usr/local/opt/postgresql/lib/libpq.5.dylib
  Referenced from: /Users/abhishiv/work/gratico/gratico/./bin/postgrest
  Reason: image not found
```

**Comment by steve-chavez** [comment]
Try installing postgrest with homebrew as well:

```
brew install postgrest
```

Ref: https://postgrest.org/en/v7.0.0/install.html

**Comment by abhishiv** [comment]
Hey @steve-chavez 

Yeah but unfortunately I think the issue is that I'm on a m1 mac. So postgres is at `/opt/homebrew/Cellar/postgresql` instead of `/usr/local/opt/postgresql` that postgrest is expecting?

Also tried building via source and that also failed due to similar issue I guess.

**Comment by jonasstenberg** [comment]
I'm in the same boat. Would love to have a pre-built ARM binary for my Macbook Pro M1.

**Comment by quinwoods** [comment]
[I made an article on how to do this!](https://quindarius.medium.com/how-to-install-postrest-on-macos-m1-67681cf5e4ec)

**Comment by quinwoods** [comment]
> [I made an article on how to do this!](https://quindarius.medium.com/how-to-install-postrest-on-macos-m1-67681cf5e4ec)

well not how to do the pre built binary but to build it from source on macos with the m1 chip

**Comment by kevcodez** [comment]
Just ran into the same issues, the docker image also does not work on my M1 mac.

**Comment by kevcodez** [comment]
With the help of Steve I managed to make it work. Downloaded the nightly Linux aarch 64 build.

Used this Dockerfile to build a custom image

```
 FROM arm64v8/ubuntu:focal
 
 RUN apt-get update -y
 RUN apt install libpq-dev zlib1g-dev jq gcc -y
 
 COPY postgrest /usr/bin/postgrest
 
 COPY postgrest.conf /etc/postgrest.conf
 
 EXPOSE 3000
 
 USER 1000
 
 CMD ["postgrest", "/etc/postgrest.conf"]
```

**Comment by yusufpapurcu** [comment]
As a note. Use latest nightly aarch build. 2020 linux releases not working.

**Comment by soedirgo** [comment]
Tested the 8.0.0_1 bottle. Works on my machine.

**Comment by saxomagic** [comment]
Hi - closed #1979 because it is a duplicate, but not because it has been resolved - just making sure there is no misunderstanding. Would very much appreciate an arm64 M1-compatible docker image built with Nix. Also, documentation is not clear on how to proceed with the Nix build using the Nix docker image for running the nix-shell for build and dev, so perhaps a bit more there on how to actually get the Nix-built docker image of PostgREST out into DockerHub? Apologies if that part is self study / common knowledge, but having build docker images with docker build and pushed to DockerHub before, this method with Nix is quite a bit different.
Btw, the documentation is really awesome in general, so my compliments to all the effort that has gone into that - learned a lot just by reading it - thanks much.

**Comment by steve-chavez** [comment]
There's now a PostgREST ARM image for [v9.0.0](https://hub.docker.com/layers/postgrest/postgrest/v9.0.0/images/sha256-564c94132abf861adfd46a9473b90e4cf06881814799490660022e1c780eaa02) and [latest](https://hub.docker.com/layers/postgrest/postgrest/latest/images/sha256-564c94132abf861adfd46a9473b90e4cf06881814799490660022e1c780eaa02?context=explore).
